spark/src/main/scala/ai/chronon/spark/SparkSessionBuilder.scala

```diff
@@ -26,9 +26,10 @@ import scala.reflect.io.Path
 import scala.util.Properties
 
 object SparkSessionBuilder {
-  @transient lazy val logger = LoggerFactory.getLogger(getClass)
+  @transient private lazy val logger = LoggerFactory.getLogger(getClass)
 
-  val DefaultWarehouseDir = new File("/tmp/chronon/spark-warehouse")
+  private val warehouseId = java.util.UUID.randomUUID().toString.takeRight(6)
```

Comment:
would it solve the potential racing condition in the CI test?

Reply:
I think it does!